### ISSUE TITLE
Improve the where clause

### ISSUE THREAD

**Issue by sharovik** [sharovik/orm#6]
Need to improve the WHERE clause generation. There should be a possibility to have a combined condition like
```
WHERE (test_id = 1 AND test_text = "test") OR (test_id = 2)
```

**Comment by sharovik** [comment]
The solution which I want to implement will be next:
In the where struct https://github.com/sharovik/orm/blob/e75feb55dfcce8f53ebb3a407f065cdfab99d2cb/query/where.go#L12 
`First` and `Second` we already have the type of interface. That means I can easily use values type of `query.Where`.

Then
When I generate the WHERE clause string
https://github.com/sharovik/orm/blob/e75feb55dfcce8f53ebb3a407f065cdfab99d2cb/clients/base_functions.go#L68
I can check the type of the field and if it is the `query.Where`, then I can run a recursive method, which will generate the child where clause string.

**Comment by sharovik** [comment]
Released a fix for that in pull-request #7
